source/extensions/transport_sockets/tls/cert_validator/spiffe/spiffe_validator.cc

```diff
@@ -30,10 +33,128 @@ namespace Tls {
 
 using SPIFFEConfig = envoy::extensions::transport_sockets::tls::v3::SPIFFECertValidatorConfig;
 
+std::shared_ptr<SpiffeData>
+SPIFFEValidator::parseTrustBundles(const std::string& trust_bundle_mapping_str) {
```

Comment:
make the return type be `absl::StatusOr<std::shared_ptr<SpiffeData>>` where the shared pointer never be nullptr if status is ok.